docs/SmartStore.md

```diff
@@ -0,0 +1,233 @@
+# SmartStore Resource Guide
+
+The Splunk Operator includes support for configuring a SmartStore remote storage volume with index support using a custom resource. The SmartStore integration is not implemented as a StorageClass. This feature and its settings rely on support integrated into Splunk Enterprise. See [SmartStore](https://docs.splunk.com/Documentation/Splunk/latest/Indexer/AboutSmartStore) for information on the feature and implementation considerations.
+
+ * SmartStore support in Kubernetes Operator is limited to Amazon S3 only
+ * You can only define SmartStore-enabled indexes using this specification
+ * You can not convert an existing index into a SmartStore-enabled index using this specification
+ * SmartStore configuration is supported on these Custom Resources: Standalone, and ClusterMaster
+
+
+SmartStore configuration involves indexes, volumes, and the volume credentials. Volume credentials are supplied through a Kubernetes secret object, where as the indexes and volume configurations are provided through the Custom Resource specification. For the CR SmartStore volume spec, the K8 secret object(`secretRef`) is where the volume keys are stored.
+
+
+Here is an example command to encode and load your remote storage volume secret key and access key in the kubernetes secret object: `kubectl create secret generic <secret_store_obj> --from-literal='s3_access_key=<access_key>' --from-literal='s3_secret_key=<secret_key>'`
+  
+
+## Creating a Standalone instance utilizing SmartStore 
```

Comment:
Should the heading read as:
"Creating a SmartStore-enabled Standalone instance"